File: pizza-wahala/pizza_app.py
import math
pizza_type = input("Enter Pizza Type: ")
guests = int(input("Enter Number of Guests: "))

boxes_of_pizza = 0
slices_leftover = 0
price = 0

if guests >= 0:

	if pizza_type.lower() == "sapa size":

		number_of_slices = 4
		price_per_box = 2000

		boxes_of_pizza = math.ceil(guests / number_of_slices)
		slices_leftover = number_of_slices - guests % number_of_slices
		price = price_per_box * boxes_of_pizza

	elif pizza_type.lower() == "small money":

		number_of_slices = 6
		price_per_box = 2400

		boxes_of_pizza = math.ceil(guests / number_of_slices)
		slices_leftover = number_of_slices - guests % number_of_slices
		price = price_per_box * boxes_of_pizza

	elif pizza_type.lower() == "big boys":

		number_of_slices = 8
		price_per_box = 3000

		boxes_of_pizza = math.ceil(guests / number_of_slices)
		slices_leftover = number_of_slices - guests % number_of_slices
		price = price_per_box * boxes_of_pizza

	elif pizza_type.lower() == "odogwu":

		number_of_slices = 12
		price_per_box = 4200

		boxes_of_pizza = math.ceil(guests / number_of_slices)
		slices_leftover = number_of_slices - guests % number_of_slices
		price = price_per_box * boxes_of_pizza

	else:

		print("Invalid Pizza Type Oga")

else:
	print("Why are you putting negative number?")

print("Number of boxes of pizza to buy = ", boxes_of_pizza)
print("Number of left over slices after serving = ", slices_leftover)
print("Prices = ", price)

File: pizza-wahala/PizzaApp.java
import java.util.Scanner;

public class PizzaApp{

	public static void main(String... args){

		Scanner input = new Scanner(System.in);
		
		System.out.print("Enter Pizza Type: ");
		String pizzaType = input.nextLine();	
		System.out.print("Enter number of Guests: ");
		int guests = input.nextInt();

//		System.out.print("Enter Pizza Type: ");
//		String pizzaType = input.nextLine();	

		int numberOfSlices;
		int pricePerBox;
		double boxesOfPizza = 0;
		int slicesLeftover = 0;
		double price = 0;
		double division = 0;
	
		if(pizzaType.equalsIgnoreCase("Sapa Size")){

			numberOfSlices = 4;
			pricePerBox = 2000;

			boxesOfPizza = Math.round(guests / numberOfSlices);
			slicesLeftover = numberOfSlices - guests % numberOfSlices;
			price = pricePerBox * boxesOfPizza;

		}else if(pizzaType.equalsIgnoreCase("Small Money")){

			numberOfSlices = 6;
			pricePerBox = 2400;

			boxesOfPizza = Math.round(guests / numberOfSlices);
			slicesLeftover = numberOfSlices - guests % numberOfSlices;
			price = pricePerBox * boxesOfPizza;

		}else if(pizzaType.equalsIgnoreCase("Big Boys")){

			numberOfSlices = 8;
			pricePerBox = 3000;

			boxesOfPizza = Math.round(guests / numberOfSlices);
			slicesLeftover = numberOfSlices - guests % numberOfSlices;
			price = pricePerBox * boxesOfPizza;

		}else if(pizzaType.equalsIgnoreCase("Odogwu")){

			numberOfSlices = 12;
			pricePerBox = 4200;

			division = guests / numberOfSlices;
			boxesOfPizza = Math.ceil(division);
			slicesLeftover = numberOfSlices - guests % numberOfSlices;
			price = pricePerBox * boxesOfPizza;

		}else{

			System.out.println("Invalid Pizza Type Oga");

		}


		System.out.println("Number of boxes of pizza to buy = " + boxesOfPizza);
		System.out.println("Number of left over slices after serving = " + slicesLeftover);
		System.out.println("Prices = " + price);



	}




}